src/Angular.js

```diff
@@ -1031,6 +1031,24 @@ function copy(source, destination, maxDepth) {
 
       case '[object Blob]':
         return new source.constructor([source], {type: source.type});
+      case '[object Map]':
+        // If we're in this case we know the environment supports Map
+        /* eslint-disable no-undef */
+        var copiedMap = new Map();
```

Comment:
I had originally used new source.constructor() here, tried changing it to Map/Set to deal /w the IE11 issue that is failing the build.  My next update I will change it back